pkg/resources/management.cattle.io/v3/project/mutator.go

```diff
@@ -126,8 +126,9 @@ func (m *Mutator) createProjectNamespace(project *v3.Project) (*v3.Project, erro
 	// for the backing namespace, we need to generate it ourselves
 	if project.Name == "" {
 		// If err is nil, (meaning "project exists", see below) we need to repeat the generation process to find a project name and backing namespace that isn't taken
+		var newName string
 		for err == nil {
-			newName := names.SimpleNameGenerator.GenerateName(project.GenerateName)
+			newName = names.SimpleNameGenerator.GenerateName(project.GenerateName)
```

Comment:
I'd probably just assign it directly instead and use it in code that follows
```suggestion
			newProject.Name = names.SimpleNameGenerator.GenerateName(project.GenerateName)
```